cost_index/urls.py

```diff
@@ -0,0 +1,21 @@
+from django.urls import path
+from rest_framework.routers import DefaultRouter
+
+from cost_index.api.views import CostIndexViewSet
+
+router = DefaultRouter()
+
+urlpatterns = [
+    path(
+        "sales/cost_indexes/",
+        CostIndexViewSet.as_view({"get": "list", "post": "create"}),
+        name="cost-index-list-create",
+    ),
+    path(
+        "sales/cost_indexes/<int:pk>",
+        CostIndexViewSet.as_view(
+            {"get": "retrieve", "put": "update", "delete": "destroy"}
+        ),
+        name="cost-index-retrieve-update-destroy",
+    ),
+]
```

Comment:
Hmm, that `router` seems to be unused there? Also, couldn't this be like
```suggestion
router = DefaultRouter()

router.register(
    r"sales/cost_indexes",
    CostIndexViewSet,
    basename="sales-cost-index",
)

urlpatterns = [
    path("", include(router.urls)),
]
```
and if needed, PATCH disabled with `ModelViewSet.http_method_names`, like it is done for example in [CustomerViewSet](https://github.com/City-of-Helsinki/apartment-application-service/blob/develop/customer/api/sales/views.py#L15)?